### ISSUE TITLE
Re-add unliftio-streams, peregrin

### ISSUE THREAD

**Issue by BardurArantsson** [commercialhaskell/lts-haskell#318]
Please re-add

[ ] unliftio-streams (cur. version 0.1.1.1)
[ ] peregrin (cur. version 0.3.1, do note that tests require an external service, so those won't run per the instructions below)

I may be hopelessly confused about how Stackage works, but looking at the lts-17 branch in the stackage main repo, it looks like the packages *should* be enabled for the LTS, but they definitely aren't showing up in the LTS -- not when using stack directly, nor on the site. So, I'm not sure if something else is wrong or it's just that some additional manual steps are needed?

Regards,

--
Checklist for adding a package:

- [x] On your machine, in a new directory, you have successfully run the following set of commands (replacing $package and $version with the name and version of the package you want to add to Stackage LTS):

      stack unpack $package-$version
      cd $package-$version
      stack init --resolver lts
      stack --resolver lts build --haddock --test --bench --no-run-benchmarks
- [x] You're using the latest version of Stack. (Otherwise please try running `stack upgrade` and re-running your commands. If that fails, please include the output of `stack --version` in this issue.)

**Comment by juhp** [comment]
Yeah it is is a little confusing: you can safely ignore the stackage branches,
"unfortunately" we don't use them for building LTS - the branches are only used for the docker image basically.

Better way to check is to look at ["See all snapshots unliftio-streams appears in](https://www.stackage.org/package/unliftio-streams/snapshots)

**Comment by juhp** [comment]
I was able to get an lts release out this week alas :(

**Comment by BardurArantsson** [comment]
No worries -- there's no rush on my end! :)

**Comment by BardurArantsson** [comment]
Ping?

**Comment by mihaimaruseac** [comment]
We didn't get an LTS last week. I'll try to get one this week, but it seems there is a bug with the process where newly added packages are dropped. So, let's see.

**Comment by BardurArantsson** [comment]
Cool, thanks. Just wanted to make sure this wasn't forgotten about :smile:

**Comment by alexeyzab** [comment]
Added in [LTS-17.13](https://www.stackage.org/lts-17.13): [unliftio-streams-0.1.1.1](https://www.stackage.org/lts-17.13/package/unliftio-streams-0.1.1.1), [peregrin-0.3.1](https://www.stackage.org/lts-17.13/package/peregrin-0.3.1)

**Comment by BardurArantsson** [comment]
Excellent, thank you very much!
